cookbooks/muparser_temperature_example/muparser-temperature-example.prm

```diff
@@ -9,7 +9,7 @@ set Dimension                              = 2
 set Use years in output instead of seconds = true
 set Start time                             = 0
 set End time                               = 0
-set Output directory                       = output-muparser-temperature-example
+set Output directory                       = output-muparser_temperature-example
```

Comment:
your script only found one dash out of three, do you want to investigate why or just reapply your command a few times?